docker-storage-setup.sh

```diff
@@ -435,11 +438,11 @@ setup_lvm_thin_pool () {
   tpool=`get_configured_thin_pool`
 
   if [ -n "$tpool" ]; then
-     local escaped_pool_lv_name=`echo $POOL_LV_NAME | sed 's/-/--/g'`
+     local escaped_pool_lv_name=`echo $CONTAINER_THINPOOL | sed 's/-/--/g'`
```

Comment:
If $argv1 is not specifyied CONTAINER_THINPOOL will be docker-root-lv.  Only way this would be a problem would be if the user specified a service file without CONTAINER_THINPOOL specified.